python/lsst/drp/tasks/assemble_coadd.py

```diff
@@ -1415,6 +1428,9 @@ class CompareWarpAssembleCoaddTask(AssembleCoaddTask):
     ConfigClass = CompareWarpAssembleCoaddConfig
     _DefaultName = "compareWarpAssembleCoadd"
 
+    _doUsePsfMatchedPolygons: bool = True
```

Comment:
Why are you re-defining the attribute instead of just setting `self._doUsePsfMatchedPolygons = True` in `__init__`?

Reply:
I'd like to be able to get this class attribute without necessarily having to create an instance - i.e., I'd like `CompareWarpAssembleCoaddTask._doUsePsfMatchedPolygons` to already return True. The only minor downside is having to define the docstring twice - which is mostly to please the editor that Sphinx. If we go the read-only class property route, I know the docstring inheritance works for sure.

Reply:
I see. It seems like you want a truly static and immutable default (which a classmethod property is good for) but I'm not so clear on whether you want a variable class/instance attribute. Does it make sense to turn this off for `CompareWarpAssembleCoaddTask`? If so then I think you want both. Likewise for whether it ought to be a class or instance attribute, I'd say the latter is generally safer even if we don't tend to make two instances of the same pipelineTask very often.

Reply:
I am considering this to be on par with the `_DefaultName` just above, which should just be treated as a constant for this class. Turning it on/off in either task would run fine - changing from the default values is just not the correct thing to do given the algorithm(s).